raco/myrial/query_tests.py

```diff
@@ -600,23 +600,23 @@ def test_sql_repeated(self):
         expected = collections.Counter([(x[3],) for x in self.emp_table])
         self.check_result(query, expected)
 
-    def test_limit(self):
+    def test_limit_without_orderby_assert(self):
```

Comment:
It's great that we have tests for the illegal combinations of LIMIT and ORDER BY, but we also need at least one test in this file that actually uses FakeDB to evaluate an ORDER BY + LIMIT query and checks the results.